src/users/containers/manage-user/manage-user.component.ts

```diff
@@ -0,0 +1,346 @@
+    const hasChanges = (rolesAdded.length > 0 || rolesDeleted.length > 0 || !UserRolesUtil.accessTypesMatch(this.user.userAccessTypes, this.updatedUser.userAccessTypes));
+
+    if (hasChanges) {
+      this.userStore.dispatch(new fromStore.EditUser(editUserRolesObj, this.organisationProfileIds));
+    } else {
+      this.summaryErrorsSubject.next({ isFromValid: false, items: [{ id: 'roles', message: 'You need to make a change before submitting. If you don\'t make a change, these permissions will stay the same' }], header: 'There is a problem' });
+      this.permissionErrors = { isInvalid: true, messages: ['You need to make a change before submitting. If you don\'t make a change, these permissions will stay the same'] };
+      return this.userStore.dispatch(new fromStore.EditUserFailure('You need to make a change before submitting. If you don\'t make a change, these permissions will stay the same'));
+    }
+  }
+
+  private getBackurl(userId: string): string {
+    return !!userId ? `/users/user/${userId}` : '/users';
+  }
+
+  public handleError(store: Store<any>, errorNumber: number): void {
+    const globalError = this.getGlobalError(errorNumber);
+    if (globalError) {
+      store.dispatch(new fromRoot.AddGlobalError(globalError));
+      store.dispatch(new fromRoot.Go({ path: ['service-down'] }));
+    }
+  }
+
+  public getGlobalError(error: number): GlobalError {
+    const errorMessages = this.getErrorMessages(error);
+    const globalError = {
+      header: this.getErrorHeader(error),
+      errors: errorMessages
+    };
+    return globalError;
+  }
+
+  private getErrorMessages(error: number) {
+    switch (error) {
+      case 400:
+        return [{
+          bodyText: 'to check the status of the user',
+          urlText: 'Refresh and go back',
+          url: '/users'
+        }];
+      case 404:
+        return [{
+          bodyText: 'to reactivate this account',
+          urlText: 'Get help',
+          url: '/get-help',
+          newTab: true
+        }, {
+          bodyText: null,
+          urlText: 'Go back to manage users',
+          url: '/users'
+        }];
+      case 422:
+        return [{
+          bodyText: 'User has been create but roles have not been refreshed.',
+          urlText: null,
+          url: null
+        }, {
+          bodyText: '',
+          urlText: 'Go back to manage users',
+          url: '/users'
+        }];
+      case 500:
+      default:
+        return [{
+          bodyText: 'Try again later.',
+          urlText: null,
+          url: null
+        }, {
+          bodyText: null,
+          urlText: 'Go back to manage users',
+          url: '/users'
+        }];
+    }
+  }
+
+  private getErrorHeader(error: number): string {
+    switch (error) {
+      case 400:
+        return 'Sorry, there is a problem';
+      case 404:
+        return 'Sorry, there is a problem with this account';
```

Comment:
I don't think it should say that. You'd probably get a 404 if your session timed out. It's not worth the potential confusion, so I'd consolidate all 4xx errors into "Sorry there is a problem" and all 5xx errors into "Sorry there is a problem with the service"

Reply:
Have moved all 4xx to one message and 5xx to another in commit:
https://github.com/hmcts/rpx-xui-manage-organisations/commit/f13b896f6d2d616fe1b63f082089d9c50d05af97